src/community/jms-cluster/activemqBroker/pom.xml

```diff
@@ -17,7 +17,7 @@
 
   <properties>
     <postgres.version>8.3-603.jdbc4</postgres.version>
-    <spring.version>3.2.5.RELEASE</spring.version>
+    <spring.version>5.2.21.RELEASE</spring.version>
```

Comment:
please use ``src/pom.xml`` dependency management to control version numbers

Reply:
Nonsense, this is a simplified repackaging of ActiveMQ, we cannot change its dependencies at will.